packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -743,9 +794,11 @@ class Scaffold extends StatefulWidget {
 
   /// The primary content of the scaffold.
   ///
-  /// Displayed below the app bar and behind the [floatingActionButton] and
-  /// [drawer]. To avoid the body being resized to avoid the window padding
-  /// (e.g., from the onscreen keyboard), see [resizeToAvoidBottomPadding].
+  /// Displayed below the [appBar], above the bottom of the ambient
```

Comment:
Is the [appBar] above the bottom of the MediaQueryData.viewInsets or is the [body] above the bottom of the MediaQueryData.viewInset?

Reply:
It's the latter